code/controllers/master.dm

```diff
@@ -14,6 +14,9 @@
 GLOBAL_REAL(Debugger, /datum/debugger) = new
 //This is the ABSOLUTE ONLY THING that should init globally like this
 //2019 update: the failsafe,config and Global controllers also do it
+//2023 update: the logger also does it
```

Comment:
These historical comments are dumb, change the whole thing to say "Nothing but these should be initting globally like this"

I feel like I commented this before. Did you resolve it or did I not finish it?

Reply:
Don't remember seeing a comment on that no